serialisation_conformance_test/src/test/java/org/ehrbase/webtemplate/CompositionConverterImp.java

```diff
@@ -111,18 +111,97 @@ public String convertFlatToRaw(
         .filter(e -> "video/mp4".equals(e.getValue()))
         .forEach(e -> currentValues.replace(e.getKey(), "video/H263-2000"));
 
-    // add missing terminology for medra_classification
-    if (currentValues.containsKey(
-        "drug_related_problem_report/medication_error/medra_classification|code")) {
-      currentValues.put(
-          "drug_related_problem_report/medication_error/medra_classification|terminology",
-          "terminology");
-    }
+    // add missing terminology for some flat examples
```

Comment:
@stefanspiska I am not sure that I fully understand. Why do we need to add the terminology and why it is not included in the example from better? They derive automatically from the template?

Reply:
The example file from better are simply wrong. Terminology is required and should be path of the flat file.

Reply:
I did not check if there would set a default if missing.